zeppelin-integration/src/test/java/org/apache/zeppelin/WebDriverManager.java

```diff
@@ -89,6 +90,14 @@ private WebDriver constructWebDriver(int port) {
         return null;
       }
     };
+    Supplier<WebDriver> edgeDriverSupplier = () -> {
```

Comment:
please also update the comment at line 49

Reply:
I've added it, thanks!